connectors/flink/src/main/java/io/delta/flink/sink/internal/committer/DeltaGlobalCommitter.java

```diff
@@ -126,8 +126,12 @@ public DeltaGlobalCommitter(
         this.conf = conf;
         this.rowType = rowType;
         this.mergeSchema = mergeSchema;
-        this.deltaLog = DeltaLog.forTable(conf,
-            new org.apache.hadoop.fs.Path(basePath.toUri()));
+        if (conf.getBoolean("io.delta.flink.usekernel", false)) {
```

Comment:
`io.delta.flink.useKernel` probably? that's most consistent with our other flag names. See `StandaloneHadoopConf`

Reply:
should probably also make a `DeltaFlinkHadoopConf` *internal* class to document this flag, it's semantics, what it means, etc etc